package.json

```diff
@@ -24,22 +24,30 @@
   "devDependencies": {
     "apollo-cache-inmemory": "^1.2.10",
     "apollo-client": "^2.4.2",
-    "apollo-link-http": "^1.5.4",
+    "apollo-link-http": "^1.5.5",
     "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-plugin-transform-runtime": "^6.23.0",
     "babel-preset-env": "^1.7.0",
     "babel-preset-react": "^6.24.1",
+    "bootstrap": "^4.1.3",
```

Comment:
does a parent's package.json auto-install the imports in it's children's packages?